woocommerce/woocommerce-ios
# [Order Fulfillment][Part 5] Mark Order Complete footer view for Review Order screen

Part 5 of #4136 

# Description
This is the final part for bringing back the order fulfillment feature! 😅 The PR aims to update Review Order screen with the Mark Order Complete button and a text to inform that user will receive an email upon marking order completed.

# Solution
- Both the Mark Order Complete button and the email label are added in a view, which is set as the footer of the table view.
- The view controller was updated to accept a block for marking order complete. When the action button is tapped, this block is triggered to ask the Order Detail screen to start the fulfillment process. A small delay was added before popping the Review Order view controller to make sure the fulfillment process is triggered before the view controller is deinit'ed.
- Removed feature flag for review order; will update release notes when I'm more certain if this PR can be merged in time for 7.1.

# Demo
<img src="https://user-images.githubusercontent.com/5533851/124713134-5a356000-df2a-11eb-8de7-25f47a906e71.gif" width=400 />

# Testing
- Navigate to Orders tab
- Select an order with In Progress status
- Select Mark Order Complete
- You should then be navigated to Review Order screen
- Scroll down to bottom, you should see a text "The customer will receive an email once order is completed" and an action button Mark Order Complete.
- Upon tapping Mark Order Complete, a toast will show up saying order is fulfilled and after a beat the Review Order will be popped.

# Update release notes:
